technical-reference.adoc

```diff
@@ -150,6 +150,19 @@ Uses a token to authenticate the connection.
 
 Uses a custom internal authentication mechanism to connect.
 
+This is the default authentication method for SaaS instances. 
+
+In order to properly authenticate using this method follow these steps:
+
+. Create an API account at Users > API Accounts.
+. Copy the client ID and account password. You will not be able to access the account password after account creation.
+. Use your client ID in the Client ID field.
+. Use your account password in the Client Secret field
+. For Auth Server, use the domain of the authentication server, i.e., login.hyperscience.net.
```

Comment:
```suggestion
. In the **Auth Server** field, enter the domain of the authentication server (`login.hyperscience.net`).
```

Reply:
I'm assuming they would only ever enter `login.hyperscience.net` here? Let me know if I'm wrong about that.

Reply:
Ah OK — thanks for clarifying!

Let's use this instead, then: 

. In the **Auth Server** field, enter the domain of the authentication server for your instance (e.g., `login.hyperscience.net`).

Reply:
Done